### ISSUE TITLE
Migrate marketing content to keystatic

### ISSUE THREAD

**Issue by jmduke** [buttondown/roadmap#1641]
(This is boring, I know, but easy intro task to the marketing codebase.)

## TL;DR

We've started migrating away from MDX in favor of [Keystatic](https://keystatic.com/), a static CMS. Let's migrate the _stories_ pages (under `/marketing/pages/stories/content`) onto them!

## Background

So, a couple things:

- Buttondown's marketing site (in `/marketing`) is a pretty standard Next application. It's got a lot of prose-y content; blog posts, case studies, and so on.
- I've been slowly moving over from using MDX for this prose to Keystatic! This is largely to make it easier for non-technical folks to make edits and changes without relying on me, and also has some benefits around routing, performance, and type safety.
- You can see some of the existing keystatic content by going to `/keystatic` in your browser to see the UI (or going to `keystatic.config.ts` to see more.)

There's a _lot_ of content that needs migration, but for now I'd only like you to focus on one thing: **case studies** (or "stories" as I call them.) You can find them [here](https://buttondown.email/stories/cassidoo) as an example.) 

This can be broken down into a couple steps:

- [ ] Create a keystatic _schema_ for these pages.
- [ ] Port over a single page, making sure the schema works.
- [ ] Create a new version of the page that displays this content (`/marketing/pages/stories/[slug].tsx`) and make sure the design looks the same
- [ ] Port over all the remaining stories

## Further content needing migration (don't worry about this for now, just book-keeping)
- [x] Competitors
- [x] Features
- [x] Use cases
- [ ] Integrations
- [x] Blog
- [x] Changelog

**Comment by smonette** [comment]
MDX has been unshipped! 🚀
